octobot_trading/personal_data/portfolios/portfolio_value_holder.py

```diff
@@ -241,18 +250,39 @@ def _recompute_origin_portfolio_initial_value(self):
 
     def _try_get_value_of_currency(self, currency, quantity, raise_error):
         """
-        try_get_value_of_currency will try to obtain the current value of the currency quantity in reference currency.
-        It will try to create the symbol that fit with the exchange logic.
+        try_get_value_of_currency will try to get the value of the given currency quantity in reference market.
+        It will try to get it from a trading pair that fit with the exchange availability.
         :return: the value found of this currency quantity, if not found returns 0.
         """
+        settlement_asset = self.portfolio_manager.reference_market \
```

Comment:
Couldn't it be handled by a "future" thing. I was just wondering if we should mix future and non future code here